### PR TITLE
Changes needed for image filters

### PR DESCRIPTION
**Pull request by feher-melos**

### CODE REVIEW

File: photoeditor/src/main/java/ja/burhanrashid52/photoeditor/ZedgeImageFilterView.java [hunk 1]
```diff
@@ -0,0 +1,209 @@
+package ja.burhanrashid52.photoeditor;
```
Comment (perezfer) [inline photoeditor/src/main/java/ja/burhanrashid52/photoeditor/ZedgeImageFilterView.java]: I think it would be a good idea to move this class into the package net.zedge.photoeditor,

File: photoeditor/src/main/java/ja/burhanrashid52/photoeditor/PhotoEditorView.java [hunk 2]
```diff
@@ -35,7 +35,7 @@
 
     private FilterImageView mImgSource;
     private BrushDrawingView mBrushDrawingView;
-    private ImageFilterView mImageFilterView;
+    private ZedgeImageFilterView mImageFilterView;
```
Comment (perezfer) [inline photoeditor/src/main/java/ja/burhanrashid52/photoeditor/PhotoEditorView.java]: Why do we need this? ZedgeImageFilterView extends ImageFilterView.

File: photoeditor/src/main/java/ja/burhanrashid52/photoeditor/PhotoEditorView.java [hunk 3]
```diff
@@ -89,7 +89,7 @@ private void init(@Nullable AttributeSet attrs) {
         brushParam.addRule(RelativeLayout.ALIGN_BOTTOM, imgSrcId);
 
         //Setup GLSurface attributes
-        mImageFilterView = new ImageFilterView(getContext());
+        mImageFilterView = new ZedgeImageFilterView(getContext());
```
Comment (perezfer) [inline photoeditor/src/main/java/ja/burhanrashid52/photoeditor/PhotoEditorView.java]: If it is not too difficult I would first try to move this to a subclass of PhotoEditorView (like ZedgePhotoEditorView) that overrides the constructor to instantiate a ZedgeImageFilterView. It is too much of  a hassle leave it like this.

File: photoeditor/src/main/java/ja/burhanrashid52/photoeditor/TextureRenderer.java [hunk 4]
```diff
@@ -65,7 +65,18 @@
 
     private static final int FLOAT_SIZE_BYTES = 4;
 
+    private boolean mIsInitialized = false;
```
Comment (perezfer) [inline photoeditor/src/main/java/ja/burhanrashid52/photoeditor/TextureRenderer.java]: Is this something that could be sent as a separate PR to the origin?

Comment (feher-melos) [reply photoeditor/src/main/java/ja/burhanrashid52/photoeditor/TextureRenderer.java]: Yes, it's a fix. Maybe can be upstreamed.

File: photoeditor/src/main/java/ja/burhanrashid52/photoeditor/PhotoEditorView.java (hunk 2 above)
Comment (feher-melos) [reply photoeditor/src/main/java/ja/burhanrashid52/photoeditor/PhotoEditorView.java]: Because I added new methods to it that are used here.
